cmd/geth/config.go

```diff
@@ -113,7 +111,6 @@ func makeConfigNode(ctx *cli.Context) (*node.Node, gethConfig) {
 		Eth:       eth.DefaultConfig,
 		Shh:       whisper.DefaultConfig,
 		Node:      defaultNodeConfig(),
```

Comment:
gofmt :)

Reply:
Damn, third time today. Not sure what's up with my editor.